File: Insurance-2306226864-fe/src/enums/service.enum.ts
export enum ServiceEnum {
  ACCOMMODATION = 'ACCOMMODATION',
  FLIGHT = 'FLIGHT',
  PACKAGE = 'PACKAGE',
  RENTALS = 'RENTALS'
}

export const ServiceLabels: Record<ServiceEnum, string> = {
  [ServiceEnum.ACCOMMODATION]: 'Accommodation',
  [ServiceEnum.FLIGHT]: 'Flight',
  [ServiceEnum.PACKAGE]: 'Package',
  [ServiceEnum.RENTALS]: 'Rentals'
}

File: Insurance-2306226864-be/src/main/java/apap/ti/_5/Insurance_2306226864_be/enums/ServiceEnum.java
package apap.ti._5.Insurance_2306226864_be.enums;

public enum ServiceEnum {
    ACCOMMODATION("Accommodation"),
    FLIGHT("Flight"),
    PACKAGE("Package"),
    RENTALS("Rentals");

    private final String displayName;

    ServiceEnum(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }

    @Override
    public String toString() {
        return displayName;
    }
}